Tests/PostgresNIOTests/New/Extensions/PSQLBackendMessageEncoder.swift

```diff
@@ -0,0 +1,279 @@
+import NIOCore
+@testable import PostgresNIO
+
+struct PSQLBackendMessageEncoder: MessageToByteEncoder {
+    typealias OutboundIn = PSQLBackendMessage
+
+    /// Called once there is data to encode.
+    ///
+    /// - parameters:
+    ///     - data: The data to encode into a `ByteBuffer`.
+    ///     - out: The `ByteBuffer` into which we want to encode.
+    func encode(data message: PSQLBackendMessage, out buffer: inout ByteBuffer) throws {
+        switch message {
+        case .authentication(let authentication):
+            buffer.writeBackendMessageID(message.id)
```

Comment:
There is nothing specifically wrong with having it this way, but it nags at me to have the same line repeated in (almost) every single `case` like this over and over. Is there any not-horrible way to factor it out? (I can't think of something off the top of my head, and it's certainly not worth spending any amount of time on.)

Reply:
Transformed into one liners.